app/scripts/components/common/rechart-chart/legend.tsx

```diff
@@ -0,0 +1,72 @@
+import React from 'react';
+import styled from 'styled-components';
+import { glsp, themeVal } from '@devseed-ui/theme-provider';
+import { LegendProps, LineChart } from 'recharts/types';
+
+import { ListItem } from './tooltip';
+import { highlightColorThemeValue } from './constant';
+
+// NEED HELP: getting error: 'LineChart' refers to a value, but is being used as a type here. Did you mean 'typeof LineChart'
+interface ReferenceLegendComponentProps extends LineChart {
```

Comment:
Can I get your help @danielfdsilva  ?
This is the `component` prop for  of  CustomizedComponent(https://recharts.org/en-US/api/Customized). I am generally not sure how to type this.

Reply:
This one was not simple. Should be easier, but even looking through the source code, I did not see other options.
If you do `interface ReferenceLegendComponentProps extends CategoricalChartProps {` it works since `CategoricalChartProps` is the base for the Charts.

You'll need to import it from `import { CategoricalChartProps } from 'recharts/types/chart/generateCategoricalChart';`